application/views/portfolio_view.php

```diff
@@ -1,51 +1,29 @@
 <?php
   session_start();
-  
+
   if (isset($_SESSION['login_message'])) {
-    echo htmlspecialchars($_SESSION['login_message']);
+        echo htmlspecialchars($_SESSION['login_message']);
     // unset($_SESSION['login_message']);
   }
 ?>
 <form onsubmit="event.preventDefault();" 
     action="/comment/store" method="POST">
-
-
-
-    
-
-    <?php 
-    if (array_key_exists('user_email', $_SESSION)) {
-        echo 'значення user_name знайдено';
-    } else {
-        echo 'NO!';
-    }
-    if ($_SESSION['user_email'] !== false):
-        
-    ?>
-
-    <label for="email">Email:</label>
-    <input type="hidden" id="email" name="email" value="<?php echo $_SESSION['user_email'] ?>" placeholder="yourEmail@gmail.com">
-
-    <?php else: ?>
-        <input type="email" id="email" name="email" value="" placeholder="yourEmail@gmail.com">
-        <?php endif; ?>
-
+    <?php if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
+        <label for="email">Email:</label>
+    <input type="hidden" name="user_email" value="<?php echo $_SESSION['user_email']; ?>" />
+<?php }; ?>
```

Comment:
Indent, psr

Reply:
removed duplicate input. left one condition.
        if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
        <label for="email">Email:</label>
        <input type="hidden" name="user_email" value="<?php echo $_SESSION['user_email']; ?>" />